lib/src/models/product.dart

```diff
@@ -167,11 +167,11 @@ class CBProductIdentifierWrapper {
 
 class CBEntitlement {
   late String subscriptionId;
-  late String featureId;
-  late String featureName;
-  late String featureType;
-  late String value;
-  late String name;
+  late String? featureId;
+  late String? featureName;
+  late String? featureType;
+  late String? value;
+  late String? name;
```

Comment:
Are all of these optional? The `featureId`, `featureName` and `featureType` are not optional AFAIK. Lets check and update it accordingly.